framework/scripts/cluster_control.py

```diff
@@ -240,6 +248,7 @@ def main():
     exclusive.add_argument('-l', '--list-nodes', action='store_const', const='list_nodes', help='List nodes')
     exclusive.add_argument('-i', '--health', action='store', nargs='?', const='health', help='Show cluster health')
     exclusive.add_argument('-u', '--usage', action='store_true', help='Show usage')
+    exclusive.add_argument('-c', '--cluster-conf', action='store_true', help=argparse.SUPPRESS)
```

Comment:
-c could be a bit confusing honestly. I think something like rc (read config) makes more sense. C or c is sometimes used to provide config, not only reading it.